src/centralserver/admin-service/core/src/main/java/org/niis/xroad/cs/admin/core/entity/ConfigurationSigningKeyEntity.java

```diff
@@ -85,6 +85,10 @@ protected ConfigurationSigningKeyEntity() {
         //JPA
     }
 
+    public ConfigurationSigningKeyEntity(int id) {
```

Comment:
Not sure if it is a good idea. We are using sequence for id. With this constructor the id must be provided to create a new entity.

Reply:
Yeah, I needed this to verify wether the private key id is passed on when transforming from `ConfigurationSigningKeyEntity` to `ConfigurationSigningKey`. But it seems this field is not really needed as we use the String `keyIdentifier` instead, therefore removed the `id` from `ConfigurationSigningKey`